packages/wallets/src/features/cfd/flows/Verification/Verification.tsx

```diff
@@ -0,0 +1,154 @@
+import React, { useMemo } from 'react';
+import { FlowProvider, TFlowProviderContext } from '../../../../components/FlowProvider';
+import { useModal } from '../../../../components/ModalProvider';
+import { ModalStepWrapper, WalletButton } from '../../../../components/Base';
+import { Loader } from '../../../../components/Loader';
+import { Onfido } from '../../screens';
+import { useAuthentication, usePOA, usePOI } from '@deriv/api';
+import { THooks } from '../../../../types';
+
+const Idv = () => {
+    return (
+        <div style={{ fontSize: 60, height: 400, width: 600 }}>
+            <h1>IDV screen</h1>
+        </div>
+    );
+};
+
+const Manual = () => {
+    return (
+        <div style={{ fontSize: 60, height: 400, width: 600 }}>
+            <h1>Manual screen</h1>
+        </div>
+    );
+};
+
+const Poa = () => {
+    return (
+        <div style={{ fontSize: 60, height: 400, width: 600 }}>
+            <h1>POA screen</h1>
+        </div>
+    );
+};
+
+const PersonalDetails = () => {
+    return (
+        <div style={{ fontSize: 60, height: 400, width: 600 }}>
+            <h1>Personal details screen</h1>
+        </div>
+    );
+};
+
+const Loading = () => {
+    return (
+        <div style={{ fontSize: 60, height: 400, width: 600 }}>
+            <Loader />
+        </div>
+    );
+};
+
+const Password = () => {
+    return <div style={{ fontSize: 60, height: 400, width: 600 }}>Password screen</div>;
+};
+
+// TODO: Replace these mock components with the screens
+const screens = {
+    idvScreen: <Idv />,
+    loadingScreen: <Loading />,
+    manualScreen: <Manual />,
+    onfidoScreen: <Onfido />,
+    passwordScreen: <Password />,
+    personalDetailsScreen: <PersonalDetails />,
+    poaScreen: <Poa />,
+};
+
+const Verification = ({ selectedJurisdiction }: { selectedJurisdiction: string }) => {
+    const { data: poiStatus, isSuccess } = usePOI();
```

Comment:
better to change this in order to consistency and readablity
```suggestion
    const { data: poiStatus, isSuccess: isSuccessPOIStatus} = usePOI();
```